packages/terra-clinical-onset-picker/tests/wdio/onset-spec.js

```diff
@@ -1,5 +1,6 @@
 const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
-const rules = { label: { enabled: false } };
+// Color contrast rule disabled due to https://github.com/cerner/terra-core/issues/1917
```

Comment:
Can we log a terra-clinical issue to remove this disabled accessibility rule once `https://github.com/cerner/terra-core/issues/1917` is resolved? 

Also, do we still need to disable the label accessibility rule? I don't think we should with the new form dependencies

Reply:
Oh do you think the form tests would be covering the accessibility testing required for this component? Yeah I think I could agree with that.

Reply:
Oh whoops misread your comment. Yeah I'll check enabling the label rule.

Reply:
It seems that the InputField is failing the label accessibility check. I'm not sure why, there is an associated label getting generated.

Reply:
hmmm @JakeLaCombe do you know what we might be missing in implementing the form input? I checked the package and there are no accessibility issues

Reply:
Seems the hidden labels prop does not add aria labels needed to pass accessibility in the form field. I'll address this by adding the proper labels and log an enhancement to terra-core to address this.